src/Cms/PageActions.php

```diff
@@ -247,6 +247,15 @@ protected function changeStatusToUnlisted()
         return $page;
     }
 
+    /**
+     * @param null $position
+     * @return self
+     */
+    public function changeSort($position = null)
+    {
+        return $this->changeStatus('listed', $position);
+    }
+
```

Comment:
As I explained in https://github.com/getkirby/kirby/issues/2475#issuecomment-593088270, I'm not 100 % happy about introducing a `changeSort()` method. Also see my comment below.

Reply:
I didn't quite understand your concerns to be honest. Having `changeNum` and `changeSort`, which do the same, doesn't sound too confusing to me. In fact, probably instead of doing `$this->changeStatus('listed', $position)` in `changeSort` it should straightforward be a simple alias. I think it will help people, when they can use `changeSort` both on `$page` and `$file` - instead of remembering that the one needs `changeNum`, the other `changeSort`.

Reply:
Although using `changeNum` wouldn't resort the siblings, right? Should `changeNum` be even public then?

Reply:
You are right – I didn't consider that `$file->changeSort()` already exists.

> Although using `changeNum` wouldn't resort the siblings, right?

Correct!

> Should `changeNum` be even public then?

There are probably use-cases where this behavior is required, for example if many pages need to be sorted in different positions in one go. It would be very slow to resort every sibling on each sort operation as each page would be sorted multiple times. So this "manual" method is useful for these use-cases.

Maybe we could make the warning in the docblock clearer though.